aten/src/ATen/native/native_functions.yaml

```diff
@@ -1802,6 +1802,16 @@
     CPU: randperm_out_cpu
     CUDA: randperm_out_cuda
 
+- func: choice(Tensor input, Tensor weights, bool replace, int k) -> Tensor
```

Comment:
I would prefer if the arguments had the same names and order as in [its inspiration in NumPy](https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.choice.html).

(With the exception of `input`, which we can leave as-is, since that's a PyTorch standard).

Reply:
I'll make this change as well.